source/jsonnet/england-wales/household/blocks/visitor/date_of_birth.jsonnet

```diff
@@ -17,6 +17,15 @@ local rules = import 'rules.libsonnet';
     type: 'General',
     answers: [
       {
+        guidance: {
+          show_guidance: 'Why we ask for visitor details',
+          hide_guidance: 'Why we ask for visitor details',
+          contents: [
+            {
+              description: 'Your answer helps to ensure that everyone is counted in the census. Add visitor details, even if you think they have been included on a census questionnaire at another address.',
+            },
+          ],
+        },
```

Comment:
Since this is identical in 3 files, it could be shared between them so that it is defined in one place.